LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright 2013-2014 Jack Naglieri <jacknagzdev@gmail.com>
```

Comment:
I'll add the LICENSE in a follow-up PR, along with CONTRIBUTING, and more.

Reply:
With the LICENSE file, it is required by FoodCritic and will fail the TravisCI build. I will just add an empty LICENSE file.